src/index.js

```diff
@@ -266,3 +266,5 @@ export {
 } from './components/PeriodDimension/utils'
 export { getRelativePeriodsOptionsById } from './components/PeriodDimension/utils/relativePeriods'
 export { getFixedPeriodsOptionsById } from './components/PeriodDimension/utils/fixedPeriods'
+
+export { default as VisualizationOptions } from './components/Options/VisualizationOptions'
```

Comment:
I see we mostly export the const directly instead of the default which needs to be aliased, any special reason for this?

Reply:
I just followed the same pattern as the other components, e.g. DataDimension and 
PeriodDimension. I think the unnamed export comes from connected components being exported in that way. But since these aren't connected I don't see why they can't be exported directly 🤷

Reply:
👍